errai-bus/src/main/java/org/jboss/errai/bus/server/servlet/AbstractErraiServlet.java

```diff
@@ -170,6 +170,7 @@ protected void sendDisconnectWithReason(OutputStream stream, final String reason
 
   protected void sendDisconnectDueToSessionExpiry(final HttpServletResponse response) throws IOException {
     response.setStatus(200);
+    response.setContentType("application/json");
```

Comment:
Is this related to this PR or is it something you found was missing on the way?

Reply:
Yeah was just something I found a long the way, was getting an error in the message bus when `sendDisconnectDueToSessionExpiry` was fired since it expects all the client bus messages as `application/json`

Reply:
FYI I had this issue as well and reported it a while back: https://issues.jboss.org/browse/ERRAI-1088

Reply:
Thanks @thebeeber, I closed both issues now :)